Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -114,6 +115,8 @@ public void Initialize(TelemetryConfiguration configuration)
             this telemetry processor will only collect for whichever instrumentation key is specified by the module in StartCollection call.
             */
 
+            this.EvaluateDisabledTrackingProperties = configuration.EvaluateExperimentalFeature("deferRequestTrackingProperties");
```

Comment:
a line/comment to describe what this experimentalfeature would be really helpful.

Reply:
I will leave this comment on the new string constant.